content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-contract.md

```diff
@@ -12,25 +12,25 @@ tags:
 title: Write contracts
```

Comment:
Did you want to change this to make it similar to the titles for states?
```suggestion
title: Write a contract
```

Reply:
Yes, good catch!